packages/seacas/applications/exodiff/ED_SystemInterface.C

```diff
@@ -30,19 +30,13 @@ namespace {
                               Tolerance &def_tol, std::vector<std::string> &names,
                               std::vector<Tolerance> &toler, int max_names);
 
-  int case_strcmp(const std::string &s1, const std::string &s2)
+  bool str_equal(const std::string &s1, const std::string &s2)
```

Comment:
Could it pay to define this in one place?

Reply:
Yes, should probably go in either IOSS or in Suplib.  Will look into that.